solutions/cf-1418C.mdx

```diff
@@ -128,5 +131,36 @@ public class MortalKombatTower {
 	}
 }
 ```
+
 </JavaSection>
+
+<PySection>
+
+```py
+for _ in range(int(input())):
+
+	n = int(input())
+	bosses = list(map(int, input().split()))
+
+	# dp[i][j] stores the minimum amount of skip pointes needed for the j-th
+	# boss when it is your turn (i = 0) and your friend's turn (i = 1).
+	dp = [[float('inf')] * (n + 1) for _ in range(2)]
+
+	dp[0][0] = 0
+
+	for j in range(len(dp[0]) - 1):
+
```

Comment:
ok so you see the comments in the c++/java code? could you port those over here as well?